src/etapi/notes.ts

```diff
@@ -223,11 +238,7 @@ function parseBoolean(obj, name) {
     return obj[name] === 'true';
 }
 
-function parseOrderDirection(obj, name) {
-    if (!(name in obj)) {
-        return undefined;
-    }
-
+function parseOrderDirection(obj: any, name: string) {
```

Comment:
Could you maybe quickly explain why you removed the if check `if (!(name in obj))`

Reply:
I've no idea. I put it back. :)